azurerm/internal/services/sql/resource_arm_sql_server.go

```diff
@@ -100,6 +101,36 @@ func resourceArmSqlServer() *schema.Resource {
 				},
 			},
 
+			"blob_extended_auditing_policy": {
```

Comment:
Given the schema is shared with the database maybe it'll make sense to write a shared schema function?